Add timer showing time since a staff member started helping a ticket

stella-lu opened the pull request:
Resolves #76. This pull requests adds a feature for staff so staff can see how long a staff member has been helping a ticket for. 

Screenshot from staff perspective:
![screen shot 2019-02-22 at 3 17 38 pm](https://user-images.githubusercontent.com/35756462/53277140-4e2d0880-36b7-11e9-89fc-6599063844d0.png)

The following screenshot was taken after a ticket was requeued. This shows that the "timer since ticket was assigned" does not show when a ticket is in the queue, even if it was previously assigned.
![screen shot 2019-02-22 at 3 34 00 pm](https://user-images.githubusercontent.com/35756462/53277145-5127f900-36b7-11e9-976b-0029b364ad50.png)

View from a student's perspective - the "time since ticket was assigned" timer does not display for the student. 
![screen shot 2019-02-22 at 3 36 01 pm](https://user-images.githubusercontent.com/35756462/53277207-9f3cfc80-36b7-11e9-9e4c-0f3d63bd7914.png)

There is only one trivial issue, in that if a ticket has been assigned and the student changes the description, this timer will update. However, it's unlikely that a student would change the description while they are already being helped.

pbitutsky commented:
I'm getting this error message when I try to run it locally. Any ideas?
```
Traceback (most recent call last):
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/eventlet/wsgi.py", line 490, in handle_one_response
    result = self.application(self.environ, start_response)
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/flask_socketio/__init__.py", line 42, in __call__
    start_response)
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/engineio/middleware.py", line 47, in __call__
    return self.engineio_app.handle_request(environ, start_response)
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/socketio/server.py", line 353, in handle_request
    return self.eio.handle_request(environ, start_response)
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/engineio/server.py", line 251, in handle_request
    transport, b64)
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/engineio/server.py", line 349, in _handle_connect
    if self._trigger_event('connect', sid, environ, async=False) is False:
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/engineio/server.py", line 378, in _trigger_event
    return self.handlers[event](*args)
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/socketio/server.py", line 494, in _handle_eio_connect
    return self._handle_connect(sid, '/')
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/socketio/server.py", line 414, in _handle_connect
    self.environ[sid]) is False:
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/socketio/server.py", line 481, in _trigger_event
    return self.handlers[namespace][event](*args)
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/flask_socketio/__init__.py", line 224, in _handler
    *args)
  File "/Users/pbitutsky/Projects/oh-queue/env/lib/python3.6/site-packages/flask_socketio/__init__.py", line 589, in _handle_event
    ret = handler()
  File "/Users/pbitutsky/Projects/oh-queue/oh_queue/views.py", line 131, in connect
    'tickets': [ticket_json(ticket) for ticket in tickets],
  File "/Users/pbitutsky/Projects/oh-queue/oh_queue/views.py", line 131, in <listcomp>
    'tickets': [ticket_json(ticket) for ticket in tickets],
  File "/Users/pbitutsky/Projects/oh-queue/oh_queue/views.py", line 36, in ticket_json
    'updated': ticket.updated.isoformat(),
AttributeError: 'NoneType' object has no attribute 'isoformat'
```

stella-lu commented:
@pbitutsky Oops, I just made a commit that changes the function `ticket_json` in the file views.py. 

The problem was that initially, ticket.updated is the NoneType object and hence `.isoformat()` could not be called on it, resulting in the error. The commit simply adds an if statement checking if `ticket.updated is None`. If it is the NoneType object, it will not return the line `'updated': ticket.updated.isoformat()`, but if it is not the NoneType object, it will return this line. 

![screen shot 2019-03-02 at 2 34 28 pm](https://user-images.githubusercontent.com/35756462/53688451-0468a180-3cf9-11e9-92d2-df15c84cfe43.png)
